packages/data/src/test/index.js

```diff
@@ -571,6 +571,7 @@ describe( 'withSelect', () => {
 		expect( wrapper.childAt( 0 ).props() ).toEqual( { foo: 'OK', propName: 'foo' } );
 
 		wrapper.setProps( { propName: 'bar' } );
+		wrapper.update();
```

Comment:
Why should we have to force a re-render? This seems quite problematic.

Reply:
I think it's due to how enzyme works, we do this in several tests. In theory it should rerender automatically when we call `setProps` but it doesn't.

Reply:
Yes, Enzyme specific thing introduced with 16.x line, it usually isn't needed, but sometimes it's the only way to make tests work :(